public/index.html

```diff
@@ -6,8 +6,8 @@
     <meta name="viewport" content="width=device-width, initial-scale=1" />
     <meta name="theme-color" content="#000000" />
     <meta
-      name="description"
-      content="Web site created using create-react-app"
+      name="tokspace"
```

Comment:
This must remain `description` so that search engines and previews will be able to find it!